neurokit2/rsp/rsp_rate.py

```diff
@@ -38,22 +35,28 @@ def rsp_rate(rsp_cleaned, peaks=None, sampling_rate=1000, window=10, hop_size=1,
     Example
     -------
     >>> import neurokit2 as nk
-    >>> rsp_signal = nk.rsp_simulate(100, respiratory_rate=15)
+    >>> rsp_signal = nk.data("rsp_200hz.txt").iloc[:,0]
+    >>> sampling_rate=200
     >>> rsp_cleaned = nk.rsp_clean(rsp_signal)
-    >>>
-    >>> rsp_rate = nk.rsp_rate(rsp_cleaned, method="peaks")
-    >>> rsp_rate2 = nk.rsp_rate(rsp_cleaned, method="xcorr")
+    >>> rsp_rate_peak = nk.rsp_rate(rsp_cleaned, sampling_rate=sampling_rate, method="peaks")
+    >>> rsp_rate_xcorr = nk.rsp_rate(rsp_cleaned, sampling_rate=sampling_rate, method="xcorr")
     """
 
     if method.lower() in ["peak", "peaks", "signal_rate"]:
         if peaks is None:
             peaks, info = rsp_peaks(rsp_cleaned, sampling_rate=sampling_rate, method=peak_method)
-        rsp_rate = signal_rate(peaks, sampling_rate=sampling_rate, desired_length=len(rsp_cleaned))
+        rsp_rate = signal_rate(peaks, sampling_rate=sampling_rate, desired_length=len(rsp_cleaned), interpolation_method="linear")
```

Comment:
I wouldn't hardcode the interpolation_method nor put linear as a default, I think we discussed it in https://github.com/neuropsychology/NeuroKit/issues/243 and opted for monotonic if I remember correctly

Reply:
Yes, you are right. I added in for comparison and didn't remove. I will add interpolation method as an option